src/main/java/at/hannibal2/skyhanni/features/garden/GardenCropsInCommand.kt

```diff
@@ -0,0 +1,66 @@
+package at.hannibal2.skyhanni.features.garden
+
+import at.hannibal2.skyhanni.features.garden.farming.CropMoneyDisplay
+import at.hannibal2.skyhanni.features.garden.farming.GardenCropSpeed.getSpeed
+import at.hannibal2.skyhanni.utils.ItemUtils.getItemName
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.sorted
+import at.hannibal2.skyhanni.utils.NEUItems
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.StringUtils.removeColor
+import at.hannibal2.skyhanni.utils.TimeUtils
+
+
+object GardenCropsInCommand {
+    private val config get() = GardenAPI.config.moneyPerHours
+
+    fun onCommand(args: Array<String>) {
+        if (!config.display) {
+            LorenzUtils.userError("shcroptime requires 'Show money per Hour' feature to be enabled to work!")
```

Comment:
Hello Hi, this is still the old command here lol

Reply:
🤦, fixed it.